plugins/mac-syphon/SyphonOBSClient.m

```diff
@@ -0,0 +1,17 @@
+#import "SyphonOBSClient.h"
+
+@implementation SyphonOBSClient
+
+- (id)initWithServerDescription:(NSDictionary<NSString *, id> *)description
+                        options:(NSDictionary<NSString *, id> *)options
+                newFrameHandler:(void (^)(SyphonOBSClient *))handler
+{
+    self = [super initWithServerDescription:description options:options newFrameHandler:handler];
```

Comment:
It is canonical to always check if the assignment was actually successful:

```
    self = [super <SOME INITIALIZER>];
    if ( self ) {
        <SOME CODE>   
    }
    return self;
```

`self` can be returned safely because `nil` objects can still be used for message passing in ObjC.